vercel.json

```diff
@@ -225,6 +225,16 @@
       "destination": "/build-decentralized-apps/reference/mainnet-risks",
       "permanent": false
     },
+    {
+      "source": "/(docs/partials/glossary/_fast-exit--liquidity-exit/?)",
+      "destination": "/(docs/partials/glossary/_fast-exit-liquidity-exit/?)",
+      "permanent": false
+    },
+    {
+      "source": "/(docs/partials/glossary/_fast-exit-liquidity-exit/?)",
+      "destination": "/(docs/partials/glossary/_fast-exit--liquidity-exit/?)",
+      "permanent": false
```

Comment:
Why is this redirect needed rather than just thde first one that's the inverse?

Reply:
This redirect is useless; I removed it. Thanks for spotting it.